automation/JJB/ruby_analysis.yaml

```diff
@@ -1,5 +1,5 @@
 - job:
-    name: sonarqube_foreman_ruby_full_analysis
+    name: sonarqube_ruby_full_analysis
```

Comment:
Slight nit: I recommend a consistant naming scheme, so this would be "sonarqube_ruby_analysis".